### ISSUE TITLE
Exception while running operation '_gui': ''utf8' codec can't decode byte 0xfa in position 136: invalid start byte'

### ISSUE THREAD

**Issue by jonfre** [bleachbit/bleachbit#630]
I get the following error when selecting a folder from File > Shred Folders > OK

_Exception while running operation '\_gui': ''utf8' codec can't decode byte 0xfa in position 136: invalid start byte'_

```
BleachBit version 2.2
Git revision 4e1f2717
GTK+ version 2.24.10
SQLite version 3.26.0
FSE = mbcs
local_cleaners_dir = None
locale_dir = C:\Users\***\AppData\Local\BleachBit\share\locale\
options_dir = C:\Users\***\AppData\Roaming\BleachBit
personal_cleaners_dir = C:\Users\***\AppData\Roaming\BleachBit\cleaners
system_cleaners_dir = C:\Users\***\AppData\Local\BleachBit\share\cleaners\
locale.getdefaultlocale = ('sv_SE', 'cp1252')
os.getenv('APPDATA') = C:\Users\***\AppData\Roaming
os.getenv('LocalAppData') = C:\Users\***\AppData\Local
os.getenv('LocalAppDataLow') = C:\Users\***\AppData\LocalLow
os.getenv('Music') = C:\Users\***\Music
os.getenv('USERPROFILE') = C:\Users\***
os.getenv('ProgramFiles') = C:\Program Files (x86)
os.getenv('ProgramW6432') = C:\Program Files
os.getenv('TMP') = C:\Users\***\AppData\Local\Temp
os.path.expanduser('~') = C:\Users\***\
platform.win32_ver[1]() = 6.1.7601
platform.platform = Windows-7-6.1.7601-SP1
platform.version = 6.1.7601
sys.argv = ['C:\\Users\\***\\AppData\\Local\\BleachBit\\bleachbit.exe', '--gui', '--no-uac']
sys.executable = C:\Users\***\AppData\Local\BleachBit\bleachbit.exe
sys.version = 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:22:17) [MSC v.1500 32 bit (Intel)]
win32com.shell.shell.IsUserAnAdmin() = True
__file__ = C:\Users\***\AppData\Local\BleachBit\library.zip\bleachbit\Diagnostic.pyo
```

NTFS file system

**Comment by az0** [comment]
It seems you are shredding a non-ASCII file that is causing issues. Lately there have been many fixes related to encoding, so please try a recent build from the [BleachBit CI site](https://ci.bleachbit.org), and let me know how it goes.

**Comment by jonfre** [comment]
Tested 2.3.1211 and it worked, thanks.

**Comment by az0** [comment]
Thanks for confirming, jonfre

**Comment by az0** [comment]
You could re-test on the newly-released [BleachBit 2.3 beta](https://www.bleachbit.org/news/bleachbit-23-beta)
